sample/mpp-library/src/commonMain/kotlin/com/icerockdev/library/SimpleViewModel.kt

```diff
@@ -6,29 +6,36 @@ package com.icerockdev.library
 
 import dev.icerock.moko.errors.ErrorEventListener
 import dev.icerock.moko.errors.handler.ExceptionHandler
-import dev.icerock.moko.errors.presenters.AlertErrorPresenter
+import dev.icerock.moko.errors.handler.PresenterExceptionHandler
+import dev.icerock.moko.errors.handler.catch
 import dev.icerock.moko.errors.mappers.ExceptionMappersStorage
 import dev.icerock.moko.errors.mappers.throwableToStringDesc
+import dev.icerock.moko.errors.presenters.AlertErrorPresenter
 import dev.icerock.moko.mvvm.dispatcher.EventsDispatcher
 import dev.icerock.moko.mvvm.livedata.LiveData
 import dev.icerock.moko.mvvm.livedata.MutableLiveData
 import dev.icerock.moko.mvvm.livedata.readOnly
 import dev.icerock.moko.mvvm.viewmodel.ViewModel
+import dev.icerock.moko.resources.desc.StringDesc
 import dev.icerock.moko.resources.desc.desc
 import kotlinx.coroutines.launch
 import kotlin.random.Random
 
-fun createSimpleViewModel(errorEventsDispatcher: EventsDispatcher<ErrorEventListener>): SimpleViewModel {
-    return SimpleViewModel(
-        exceptionHandler = ExceptionHandler(
-            errorEventsDispatcher = errorEventsDispatcher,
-            errorPresenter = AlertErrorPresenter(
-                exceptionMapper = ExceptionMappersStorage::throwableToStringDesc,
-                alertTitle = MR.strings.errorDialogTitle.desc()
-            )
-        )
+fun createSimpleViewModel(
+    errorEventsDispatcher: EventsDispatcher<ErrorEventListener<StringDesc>>
+) = SimpleViewModel(
+    exceptionHandler = PresenterExceptionHandler(
+        errorEventsDispatcher = errorEventsDispatcher,
```

Comment:
maybe set `eventsDispatcher` optional? with default value. `EventsDispatcher()` can be called in common code...
possibility to pass own dispatcher required for tests, but not for usage in app i think

Reply:
https://github.com/icerockdev/moko-mvvm/blob/master/mvvm/src/androidMain/kotlin/dev/icerock/moko/mvvm/dispatcher/EventsDispatcher.kt#L18

Reply:
`errorEventsDispatcher` default value was added to ExceptionHandler factory.